### ISSUE TITLE
Rake task isn't always setting slugs

### ISSUE THREAD

**Issue by dblock** [digitalplaywright/mongoid-slug#217]
Looks like something has changed recently with a newer version of mongoid whereas `slugs?` is returning `true` for an empty array.

```
mongoid_slug:set
  when models parameter is passed
    goes though all documents of passed models and sets slug if not already set (FAILED - 1)

Failures:

  1) mongoid_slug:set when models parameter is passed goes though all documents of passed models and sets slug if not already set
     Failure/Error: expect(uninitalized_slug1.reload.slugs).to eq(['uninitalized-slug1'])
     
       expected: ["uninitalized-slug1"]
            got: []
     
       (compared using ==)
     # ./spec/tasks/mongoid_slug_rake_spec.rb:43:in `block (3 levels) in <top (required)>'

```

**Comment by Fudoshiki** [comment]
Fixed here https://github.com/digitalplaywright/mongoid-slug/pull/219

**Comment by Fudoshiki** [comment]
@dblock We can close this

**Comment by dblock** [comment]
Closed via #219

**Comment by dblock** [comment]
FYI was released in 5.3.0
